homeassistant/components/idasen_desk/config_flow.py

```diff
@@ -64,6 +64,9 @@ async def async_step_user(
             desk = Desk(None)
             try:
                 await desk.connect(discovery_info.device, monitor_height=False)
+            except AuthFailedError as err:
+                _LOGGER.exception("AuthFailedError", exc_info=err)
+                errors["base"] = "auth_failed"
```

Comment:
This line is not covered by tests. We require 100% test coverage on our config flows.

../Frenck